examples/gno.land/r/demo/users/users.gno

```diff
@@ -323,12 +325,26 @@ func Render(path string) string {
 	}
 }
 
-func renderHome() string {
+func renderHome(path string) string {
 	doc := ""
-	name2User.Iterate("", "", func(key string, value interface{}) bool {
-		user := value.(*users.User)
+
+	page := pager.NewPager(&name2User, 50).MustGetPageByPath(path)
+
+	for _, item := range page.Items {
+		user := item.Value.(*users.User)
 		doc += " * [" + user.Name + "](/r/demo/users:" + user.Name + ")\n"
-		return false
-	})
+	}
+
+	doc += page.Selector()
 	return doc
 }
+
+func splitPathAndQuery(fullPath string) (string, string) {
+	parts := strings.SplitN(fullPath, "?", 2)
```

Comment:
When I click on page "2", the URL becomes "http://127.0.0.1:8888/r/demo/users?page=2".  But I don't think that the query string with the "?" is passed to `Render`.

Reply:
Right, they are not being passed:
https://gno.land/r/g1hy6zry03hg5d8le9s2w4fxme6236hkgd928dun/echorender:foo/bar?baz=bar

Reply:
Created #2876 in case is helpful